nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -471,12 +485,16 @@ def run(self, gpu):
                 threading.Thread(target=self._shutdown_client, args=[client]).start()
 
     def _shutdown_client(self, client):
-        client.communicator.heartbeat_done = True
-        time.sleep(3)
-        # client.terminate()
-        client.close()
-        client.status = ClientStatus.STOPPED
-        client.communicator.cell.stop()
+        try:
+            client.communicator.heartbeat_done = True
+            time.sleep(3)
```

Comment:
Why do we need to sleep 3 secs here? Is this client's job cell or parent cell?

Reply:
no need for the sleep any more